### DISCUSSION TITLE
Receiving Long SMS (more than 160 characters)

### DISCUSSION THREAD

**Discussion by anukantn** [python-smpplib/python-smpplib#168] [Q&A]
Hi, 
Thank you for the library.
I am trying to figure out how to receive long SMS messages, i.e, messages that have more than 160 characters, is there any provision for this in the library?

**Comment by eigenein** [comment]
Hi @anukantn,

Unfortunately, no. The best I can suggest at the moment – is to look into https://github.com/python-smpplib/python-smpplib/blob/master/smpplib/gsm.py and try reversing the splitting logic from there.
